src/email-sender.js

```diff
@@ -0,0 +1,39 @@
+const nodemailer = require('nodemailer');
+
+/*
+*                             HOW TO USE                                 *
+* Import this file - const sendEmail = require('./email-sender);         *
+* send email - sendEmail.sendMessage("Put your error message in here");  *
+*/
+
+exports.sendMessage = async function (errorMessage) {
+  // Recipients list of users to send the email too
+  // receipiants is commas separated (emailName@hotmail.com,emailName2@hotmail.com)
+  const receipiants = '';
+
+  // Credientials to send an email from
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: '', // Email Name (.env variables must be used refer to issue#60)
+      pass: '', // Email Pass (.env variables must be used refer to issue#60)
```

Comment:
Let's do `SMTP_SERVER`, `SMTP_USERNAME`, `SMTP_PASSWORD` as environment variables in `env.sample`, and you can read them from `process.env`